### DISCUSSION TITLE
Getting error when trying to set up  LSP

### DISCUSSION THREAD

**Discussion by amrit3128** [NvChad/NvChad#2630] [Q&A] [answered]
![image](https://github.com/NvChad/NvChad/assets/128936479/b9f45512-72be-4ed3-9563-4281cb91bab3)
![image](https://github.com/NvChad/NvChad/assets/128936479/de70d204-c8bb-4160-9091-1cac7497d71f)
![image](https://github.com/NvChad/NvChad/assets/128936479/fc08d4b6-2cb8-45b8-bbc2-77950087a2ae)

**Comment by siduck** [comment, marked as answer]
that warning is normal and doesnt harm. ignore it
